.github/workflows/release.yaml

```
@@ -2,35 +2,71 @@
+        run: |
+          VERSION=$(yq '.package.version' -p toml Cargo.toml)
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          echo "TAG=v$VERSION" >> $GITHUB_ENV
+
+      - name: Create and push tag
+        run: |
+          git tag ${{ env.TAG }}
+          git push origin ${{ env.TAG }}
+
+      - name: Set up Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.11)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)